supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -35,6 +456,149 @@ For documentation questions, contact rhel-docs@redhat.com.
 
 *See also*: xref:upgrade-rhel[upgrade], xref:clean-install[clean install]
 
+
+[discrete]
+[[instrumentation-module]]
+==== image:images/yes.png[yes] instrumentation module (noun)
+*Description*: The kernel module built from a `SystemTap` script; the `SystemTap` module is built on the host system, and will be loaded on the target kernel of the target system.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:host-system[host system], xref:target-kernel[target kernel], xref:target-system[target system]
+
+
+[discrete]
+[[inventory]]
+==== image:images/yes.png[yes] inventory (noun)
+*Description*: A list of managed nodes. An inventory file is also sometimes called a _hostfile_. Your inventory can specify information like IP address for each managed node. An inventory can also organize managed nodes, creating and nesting groups for easier scaling.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:managed-nodes[managed nodes]
+
+[discrete]
+[[kerberos-authentication-indicators]]
+==== image:images/yes.png[yes] Kerberos authentication indicators (noun)
+*Description*: Authentication indicators are attached to Kerberos tickets and represent the initial authentication method used to acquire a ticket:
+
+* `otp` for two-factor authentication (password + one-time password)
+* `radius` for Remote Authentication Dial-In User Service (RADIUS) authentication (commonly for 802.1x authentication)
+* `pkinit` for Public Key Cryptography for Initial Authentication in Kerberos (PKINIT), smart card, or certificate authentication
+* `hardened` for passwords hardened against brute-force attempts
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[kerberos-keytab]]
+==== image:images/yes.png[yes] Kerberos keytab (noun)
+*Description*: While a password is the default authentication method for a user, keytabs are the default authentication method for hosts and services. A Kerberos keytab is a file that contains a list of Kerberos principals and their associated encryption keys, so a service can retrieve its own Kerberos key and verify a user’s identity. For example, every IdM client has an `/etc/krb5.keytab` file that stores information about the `host` principal, which represents the client machine in the Kerberos realm.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos-principal[Kerberos principal]
+
+[discrete]
+[[kerberos-principal]]
+==== image:images/yes.png[yes] Kerberos principal (noun)
+*Description*:  Unique Kerberos principals identify each user, service, and host in a Kerberos realm. Kerberos principals adhere to the following naming conventions:
+
+* For users: `identifier@REALM`, such as `admin@EXAMPLE.COM`.
```

Comment:
Can you remove this period to be consistent with the other bullets?

Reply:
+1